src/mkpsxiso/cdwriter.cpp

```diff
@@ -305,6 +305,18 @@ size_t cd::IsoWriter::WriteBytesRaw(void* data, size_t bytes) {
 
 }
 
+size_t  cd::IsoWriter::WriteBlankSectors(const size_t count)
+{
+	char blank[CD_SECTOR_SIZE];
+	memset( blank, 0x00, CD_SECTOR_SIZE );

```

Comment:
```suggestion
	const char blank[CD_SECTOR_SIZE] {};
```

Nitpick, but then you can make it const (if `WriteBytesRaw` doesn't object).